spec/unit/chunker_spec.rb

```diff
@@ -69,16 +96,45 @@ def @throttler.stride
                                                            :start     => 1,
                                                            :limit     => 1)
 
+      @connection.expects(:select_value).with(regexp_matches(/where id >= 1 order by id limit 1 offset 0/)).returns(nil)
```

Comment:
Okay, I'm :+1: on this and I now understand the reasoning. I still think the mock should return 1 :) I ran the tests with `.returns(1)` and it still passes:

`90 runs, 179 assertions, 0 failures, 0 errors, 0 skips`

But yeah, with that `.min` clause in there it makes no difference at all. 🚢  it.